src/index.js

```diff
@@ -5,7 +5,7 @@ import "./assets/styles/index.css";
 import App from "./App";
 //testing api end points / remove later
 // Requests are objects of exports
-import * as Requests from "./apis/fetchRequests";
+import * as Requests from "./apis/subdued-fog-mouth";
```

Comment:
Unnecessary change. To switch to the deployed API you would change the baseUrl in fetchRequests to the glitch URL.